po/r-novice-gapminder.ja.po

```diff
@@ -543,8 +543,8 @@ msgid ""
 "  so in any reasonable manner, but not in any way that suggests the\n"
 "  licensor endorses you or your use."
 msgstr ""
-"  your work is derived from work that is Copyright © Software\n"
-"  Carpentry and, where practical, linking to http://software-carpentry.org/), \n"
+"  著作権©ソフトウェアカーペントリーである作品から派生していることに言及して、\n"
```

Comment:
```suggestion
"  ©ソフトウェアカーペントリーである著作物から派生していることを記載して、\n"
```

Reply:
This seems good to me but I'll leave it up for a second opinion. @joelnitta ?

Reply:
I've gone with this:
```
"  ソフトウェアカーペントリーの著作物 (Software Carpentry©) から派生していることを記載して、\n"
```